README.md

```diff
@@ -1,8 +1,8 @@
-# Kyma Documentation Component
+# Documentation component
```

Comment:
```suggestion
# Documentation Component
```

Reply:
why? component is not part of the name, it is technical indicator that this is a component